obvious-ci.docker/linux64_obvci/Dockerfile

```diff
@@ -2,34 +2,52 @@ FROM centos:6
 
 MAINTAINER Phil Elson <pelson.pub@gmail.com>
 
-RUN yum install -y \
+
+# Set an encoding to make things work smoothly.
+ENV LANG en_US.UTF-8
+
+# Install requirements for 64-bit only.
+RUN echo "exclude=*.i386 *.i686" >> /etc/yum.conf && \
```

Comment:
Given our discussion today. This change sounds controversial. Though we are not actually supporting 32-bit now AFAICT. That being said, will remove it anyways.